internal/slicer/slicer_test.go

```diff
@@ -65,19 +66,21 @@ var packageEntries = map[string][]testutil.TarEntry{
 	},
 }
 
-// Hardcoded copyright files from test-package package that will be automatically
-// injected into every slice.
-var copyrightEntries = map[string]string{
-	"/usr/":                                 "dir 0755",
-	"/usr/share/":                           "dir 0755",
-	"/usr/share/doc/":                       "dir 0755",
-	"/usr/share/doc/test-package/":          "dir 0755",
-	"/usr/share/doc/test-package/copyright": "file 0644 c2fca2aa",
+var testPackageCopyrightEntries = []testutil.TarEntry{
+	// Hardcoded copyright paths.
+	testutil.Dir(0755, "./usr/"),
+	testutil.Dir(0755, "./usr/share/"),
+	testutil.Dir(0755, "./usr/share/doc/"),
+	testutil.Dir(0755, "./usr/share/doc/test-package/"),
+	testutil.Reg(0644, "./usr/share/doc/test-package/copyright", "copyright"),
 }
 
 var slicerTests = []slicerTest{{
 	summary: "Basic slicing",
 	slices:  []setup.SliceKey{{"test-package", "myslice"}},
+	pkgs: map[string][]byte{
+		"test-package": testutil.PackageData["test-package"],
```

Comment:
Instead of repeating this almost everywhere, I'd suggest having it as a default (`if test.pkgs == nil ...`).